pkg/rpcserver/runner.go

```diff
@@ -122,6 +204,7 @@ func (runner *Runner) connectionLoop() error {
 }
 
 func (runner *Runner) sendRequest(req *queue.Request) error {
+	conn := runner.conn
```

Comment:
Now this change is not necessary. There is no point to cache 1 random field from Runner on stack, and not cache the reset of fields.